### DISCUSSION TITLE
web plugin `readonly` and `include_paths` not working?

### DISCUSSION THREAD

**Discussion by iveney** [beetbox/beets#5290] [Q&A]
I enabled web in the plugin list, and can successfully run it with `beet web`

```
web:
  host: 0.0.0.0
  readonly: no
  include_paths: yes
```

However, I cannot edit the tag or show the path. I can play and download without any problem. How do I enable editing and show the paths?

![image](https://github.com/beetbox/beets/assets/51033/a4224ceb-cf1b-4d83-bdd6-c53c472a9d57)

```
❯ beet --version
beets version 2.0.0
Python version 3.12.3
plugins: duplicates, edit, fetchart, fromfilename, info, inline, lastgenre, mbsubmit, mbsync, plexupdate, spotify, unimported, web

```

**Comment by DjSlash** [comment]
I tried the same configuration and got the same results. So this might be a bug.
